alembic/script.py.mako

```diff
@@ -6,6 +6,7 @@ Revises: ${down_revision | comma,n}
 Create Date: ${create_date}
 
 """
+import fastapi_users_db_sqlalchemy as fastapi_users_db_sqlalchemy
```

Comment:
Is this import being used?

Reply:
Yes! So this is a `.mako` and not an actual `.python` file. It's the template alembic uses when creating new migrations. If you look at the code generated file [alembic/versions/684501e372d2_adding_oauth.py](https://github.com/Glitched/ReservAI/pull/10/files/14a66331ad350bc0f5089c8fe0cd5bdc17c67e17#diff-aa91b51ee1aeec534d5302f6809e0a497cee88ad5efe86036cf6e62c4805f33d), you can see that it does reference fastapi_users_db_sqlalchemy

Reply:
Ah I see, I'll read up a bit more on `alembic` to get up to speed. Thanks!